### DISCUSSION TITLE
Drop .NET Framework 4.x support

### DISCUSSION THREAD

**Discussion by Havunen** [empira/PDFsharp#194] [Q&A] [answered]
Hi,

Have you considered dropping .NET 4.X / netstandard support ?

Maybe it could be left somewhere to different branch or something and stated that only critical security updates will be included and new features are developed only for modern target frameworks?

Supporting that old framework makes collaboration more difficult compiler directives, slow test runs etc.

**Comment by ThomasHoevel** [comment, marked as answer]
We had already removed support for .NET Framework and then we brought it back.

I think we'll stick to .NET Framework as long as Microsoft supports it.

What's the problem? You can set your application to e.g. .NET 8 only and ignore that there are DLLs for .NET Framework in the NuGet package.
